Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,564 @@
+is then use the VTKImageToImage adaptor in order to convert the
+[`vtkImageData`](https://vtk.org/doc/nightly/html/classvtkImageData.html) into an [`itk::Image`](https://itk.org/Doxygen/html/classitk_1_1Image.html).
+
+Supported VTK file formats can be found in https://docs.vtk.org/en/latest/supported_data_formats.html.
+
+### What mesh and point set file formats can ITK import and export?
+
+ITK supports reading and writing the following mesh and point set file
+formats:
+
+| File Format | Read/Write | Import |
+| ----------- | ---------- | ------ |
+| [BYU](http://www.eg-models.de/formats/Format_Byu.html) | [
+`itk::BYUMeshIO`]https://itk.org/Doxygen/html/classitk_1_1BYUMeshIO.html | |
+| [FreeSurfer surface, binary and ASCII](http://www.grahamwideman.com/gw/brain/fs/surfacefileformats.htm) | [`itk::FreeSurferBinaryMeshIO`](https://itk.org/Doxygen/html/classitk_1_1FreeSurferBinaryMeshIO.html), [`itk::FreeSurferAsciiMeshIO`](https://itk.org/Doxygen/html/classitk_1_1FreeSurferAsciiMeshIO.html) | |
+| [OFF](https://en.wikipedia.org/wiki/OFF_%28file_format%29) | [`ìtk::OFFMeshIO`](https://itk.org/Doxygen/html/classitk_1_1OFFMeshIO.html) | |
+| [STL](https://en.wikipedia.org/wiki/STL_%28file_format%29) | | |
+| [SWC Neuron Morphology](https://swc-specification.readthedocs.io/en/latest/) | | |
+| [OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file) | [`ìtk::OBJMeshIO`](https://itk.org/Doxygen/html/classitk_1_1OBJMeshIO.html) | |
+| [VTK legacy file format for vtkPolyData](https://www.vtk.org/wp-content/uploads/2015/04/file-formats.pdf) | [`itk::VTKPolyDataMeshIO`](https://itk.org/Doxygen/html/classitk_1_1VTKPolyDataMeshIO.html) | |
+
+More information on supported mesh and point set file formats is available on the
+[ITK-wasm Mesh and Point Set File Formats](https://wasm.itk.org/en/latest/introduction/file_formats/meshes.html)
+site.
+
+### What transform file formats can ITK import and export?
+
+ITK supports reading and writing the following transform file formats:
+
+| File Format | Read/Write | Import |
+| ----------- | ---------- | ------ |
+| InsightLegacy | [`itk::TxtTransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1TxtTransformIOTemplate.html) | |
+| ITK HDF5 | [`itk::HDF5TransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1HDF5TransformIOTemplate.html) | |
+| MATLAB | [`itk::MatlabTransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1MatlabTransformIOTemplate.html) | |
+| MINC (Medical Image NetCDF) | `itk::MINCTransformIOTemplate` | |
+
+#### DICOM data
+
+##### How do I read a volume from a DICOM series?
+
+The following are alternative options for reading DICOM series. The
+first one in the list is the recommended option. The others are provided
+in case of desperation.
+
+1. Use the [itk-wasm dicom](https://itk-wasm-dicom-docs.on.fleek.co/#/),
+   JavaScript pacakge, which is available from [here](https://itk-wasm-dicom-python-docs.on.fleek.co/).
```

Comment:
We can remove `, JavaScript packgae,` -- it is a C++ CLI, JavaScript package, and Python package.

Reply:
Removed in e808466.